README.md

```diff
@@ -65,6 +65,7 @@ To produce a short demonstration. More complex demos welcome!
 - Binding Author(s):
     - m4b <m4b.github.io@gmail.com>
     - Richo Healey <richo@psych0tik.net>
+    - Travis Finkenauer <tmfinken@gmail.com>
```

Comment:
Either make this alphabet or put yourself at the top :)

Reply:
I decided to keep it alphabetical (ignoring case)  =)